app/javascript/components/Header/ExtendedHeader.tsx

```diff
@@ -83,6 +83,7 @@ export const ExtendedHeader = ({ page, toggleMobileNav, mobileNavOpen, primaryHe
         {showMobileMenu && <NavMenuButton onClick={toggleMobileNav} label="Menu" />}
       </div>
       <ExtendedNav
+        aria-label="Primary navigation"
```

Comment:
Do we need to translate this?

Reply:
@stevenbarragan - good point.

@jimkreft Can you confirm if we need translations for below aria-labels we are using?

1. `aria-label="Primary navigation"`
2. `aria-label="Vertical navigation"`

Reply:
@yogesh27 yes, if it's prose, it should be pulling from the localizations YAML.

Reply:
@jimkreft One suggestion I need, I am currently using "Vertical navigation" as an aria-label for our vertical navigation links. Is this naming fine? or we can have "Secondary navigation"?

Reply:
I think secondary navigation works better